.mergify.yml

```diff
@@ -6,8 +6,6 @@ queue_rules:
       # Conditions to get out of the queue (= merged)
       - check-success=flake8
       - check-success=integration
-      - check-success=test (2.7)
-      - check-success=test (3.6)
```

Comment:
It looks like 3.6 may have accidentally be removed here?

Reply:
It also looks like 3.10 should be added.

Reply:
I think it's already there after expanding